pages/apply.html

```diff
@@ -6,6 +6,7 @@
 banner_file: banner--join-us-lg.svg
 banner_file_mobile: banner--join-us-sm.svg
 social_image: social-card--join-us.png
+social_image_alt_text: "Blue graphic reads 'USDS needs you. Apply now' with USDS logo: navy blue shield with three stars, two stripes and yellow wings"
```

Comment:
Suggested text edit:

"Graphic with bold text that reads 'USDS needs you. Apply now.' Text is displayed under the USDS logo, a navy blue shield with three stars, two stripes and yellow wings."